cmd/psa/verify_as_relyingparty.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package psa
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/veraison/apiclient/verification"
+	"github.com/veraison/evcli/common"
+	"github.com/veraison/psatoken"
+)
+
+var (
+	relyingPartyTokenFile *string
+	relyingPartyAPIURL    *string
```

Comment:
The variable name sounds slightly misleading to me!

Reply:
same comment as above.  It's the chosen convention to avoid name clashes.